boards/DIM/Src/main.c

```diff
@@ -68,14 +80,27 @@ static void MX_ADC2_Init(void);
 void        RunTask100Hz(void const *argument);
 void        RunTaskCanRx(void const *argument);
 void        RunTaskCanTx(void const *argument);
+void        StartTask1kHz(void const *argument);
 
 /* USER CODE BEGIN PFP */
 
 /* USER CODE END PFP */
 
 /* Private user code ---------------------------------------------------------*/
 /* USER CODE BEGIN 0 */
+static void CanRxQueueOverflowCallBack(size_t overflow_count)
+{
+    shared_assert(can_tx != NULL);
```

Comment:
In keeping with the discussion we had recently about checking for NULL args, is this the sort of the place we need to do it? IMO it doesn't seem critical enough.

Perhaps add it to the auto-generated code for `CanTx` and `CanRx` so we don't have to worry about it?

Reply:
Oh yeah I totally had the same thought while porting this code over. Let me clean this up for all the boards in this PR.